pb-rs/src/parser.rs

```diff
@@ -327,7 +327,7 @@ fn message(input: &str) -> IResult<&str, Message> {
                 delimited(
                     tag("{"),
                     many0(delimited(many0(br), message_event, many0(br))),
-                    tag("}"),
+                    preceded(many0(br), tag("}")),
```

Comment:
I'm not that familiar with `nom`, but I think `message_event()` already handles whitespace parsing internally, so I was thinking of using #217's solution to take advantage of that.